tests/src/main/java/org/hibernate/beanvalidation/tck/util/ConstraintViolationAssert.java

```diff
@@ -45,29 +45,37 @@
 	private ConstraintViolationAssert() {
 	}
 
+	public static ConstraintViolationSetAssert assertThat(Set<? extends ConstraintViolation<?>> actualViolations) {
+		return new ConstraintViolationSetAssert( actualViolations );
+	}
+
 	public static PathExpectation pathWith() {
 		return new PathExpectation();
 	}
 
-	public static class ConstraintViolationSetAssert {
+	public static class ConstraintViolationSetAssert extends IterableAssert<ConstraintViolation<?>> {
 
-		public static void assertContainsOnlyPaths(Set<? extends ConstraintViolation<?>> actualViolations, PathExpectation... paths) {
-			assertNotNull( paths );
+		protected ConstraintViolationSetAssert(Set<? extends ConstraintViolation<?>> actualViolations) {
+			super( actualViolations );
+		}
+
+		public void containsOnlyPaths(PathExpectation... paths) {
+			isNotNull();
 
 			List<PathExpectation> actualPaths = new ArrayList<>();
 
-			for ( ConstraintViolation<?> violation : actualViolations ) {
+			for ( ConstraintViolation<?> violation : actual ) {
 				actualPaths.add( new PathExpectation( violation.getPropertyPath() ) );
 			}
 
-			assertContainsOnly( actualPaths, paths );
+			Assertions.assertThat( actualPaths ).containsOnly( paths );
 		}
 
-		public static void assertContainsPath(Set<? extends ConstraintViolation<?>> actualViolations, PathExpectation expectedPath) {
-			assertNotNull( expectedPath );
+		public void containsPath(PathExpectation expectedPath) {
+			isNotNull();
```

Comment:
No. It tests that the actualViolations parameter passed in the assertThat is not null. (it's a straight copy of what is in HV)